pkg/kubelet/container/runtime.go

```diff
@@ -91,6 +91,8 @@ type Runtime interface {
 	ContainerCommandRunner
 	// ContainerAttach encapsulates the attaching to containers for testability
 	ContainerAttacher
+	// UpdateApiVersionCache updates api version cache of the runtime.
+	UpdateApiVersionCache() error
```

Comment:
I'd prefer the updates happening automatically within the runtime and runtime should just expose the version information.